plugins/module_utils/foreman_helper.py

```diff
@@ -26,6 +26,34 @@
     APYPIE_IMP_ERR = traceback.format_exc()
 
 
+def _entity_spec_helper(spec):
```

Comment:
Given this is pure Python code, with no relation to Ansible whatsoever, I think this should get a set of unittests :)